### ISSUE TITLE
Multiword strings

### ISSUE THREAD

**Issue by Hamcha** [Hamcha/dvx#2]
Currently broken since we split on whitespace before parsing.

Test changes by executing mulstr.dvx

**Comment by silverweed** [comment]
Separators in general don't work in strings:
`SCRIVO {CIAO:MONDO}`
will result in:

```
DvxCall (TName "SCRIVO") [
    DvxTok (TName "{CIAO"),
    DvxTok TColon,
    DvxTok (TName "MONDO}")
]
```

**Comment by Hamcha** [comment]
Solved as of defeb3b
